README.md

```diff
@@ -57,6 +57,14 @@ To trigger an automated human readable diff, add the following tag to a comment
 
 CL is used in a number of applications including: 
 
+**CellxGene**
+
+The Cell ontology is used to annotate cell types on the single cell transcriptomics platform CellxGene.  All datasets on CellXGene are annotated according to a standard schema that specifies use of CL to record Cell Type.  Uses in CellXgene tools:
+ - In the [Discover](https://cellxgene.cziscience.com/)) tool, the structure of CL is used to drive faceted searching for datasets and collections by Cell Type ,
+ - In the [Gene Expression](https://cellxgene.cziscience.com/gene-expression), CL annotation is used to aggregate expression data
+ - In [CellGuide](https://cellxgene.cziscience.com/cellguide) CL term names and synonyms drive search,  terms are used to structure the guide and ontology structure support a graphical browser
```

Comment:
I am not sure I am undestanding this sentence correctly, but as I understand it:
 - In [CellGuide](https://cellxgene.cziscience.com/cellguide), CL term names and synonyms drive search  Terms are used to structure the guide, and the ontology structure supports a graphical browser.